src/frontend/src/instance.rs

```diff
@@ -629,41 +556,33 @@ impl SqlQueryHandler for Instance {
                     Ok(Output::AffectedRows(affected))
                 }
             },
-            Statement::CreateTable(create) => {
-                let create_expr = self
-                    .create_expr_factory
-                    .create_expr_by_stmt(&create)
-                    .await
+            Statement::Alter(alter_stmt) => {
+                let expr = AlterExpr::try_from(alter_stmt)
                     .map_err(BoxedError::new)
-                    .context(server_error::ExecuteQuerySnafu { query })?;
-
-                self.handle_create_table(create_expr, create.partitions)
-                    .await
-            }
-            Statement::CreateDatabase(c) => {
-                let expr = CreateDatabaseExpr {
-                    database_name: c.name.to_string(),
+                    .context(server_error::ExecuteAlterSnafu { query })?;
+                let expr = AdminExpr {
+                    header: Some(ExprHeader {
+                        version: PROTOCOL_VERSION,
+                    }),
```

Comment:
Maybe we should add a function like `create(expr: admin_expr::Expr)` to `AdminExpr` to avoid this boilerplate. But it doesn't have to do in this PR